### PR TITLE
Fix broken links in the web app and create required view-components

### PR DESCRIPTION
**Pull request by mahendrark**
Fix #Issue

**Changes proposed in this pull request**:
- Make sure all the clickable links lead to pages with meaningful results
- Create the required view-components to describe the UI of the tool

The following steps were realized, as well (if applies):
- [ ] Apply black (`black . --exclude docs/`)
- [ ] Update the CHANGELOG.md

The following optional steps were realized:
- [ ] Use in-line comments to explain your code
- [ ] Write docstrings to your code
- [ ] For new functionalities: explain in readthedocs
- [ ] Write test(s) for your new patch of code

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl
-institut/open_plan/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

File: docs/view_components/create_energysys_components.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
+.. _create_es_comps-label:
```
Comment (Bachibouzouk) [inline docs/view_components/create_energysys_components.rst]: This is already described in energy_system_network

File: docs/view_components/load_consumption_ts.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
+.. _load_timeseries-label:
+
```
Comment (Bachibouzouk) [inline docs/view_components/load_consumption_ts.rst]: why is this needed on top of `load_input_dataseries.rst` ?

File: docs/view_components/project_list.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
+.. _project_list-label:
+
+List of Projects
```
Comment (Bachibouzouk) [inline docs/view_components/project_list.rst:3]: I would rather define "project" as an abstract component. A list is basic enough so that we don't need to explain what it is :)

The actions of edit, delete, show or select a project can also be described in the abstract component.

File: docs/view_components/scenario_list.rst [hunk 4]
```diff
@@ -0,0 +1,84 @@
+.. _scenario_list-label:
+
+List of Scenarios
```
Comment (Bachibouzouk) [inline docs/view_components/scenario_list.rst:3]: Same comment as for project list

**Review by Bachibouzouk — APPROVED** [review]
I would maybe precise what is vizualization step (you can use <p class="cell"></p> tag to write text in a separate line in any of the step view).
You can merge this and continue to create endpoints for links of the last steps
You should also implement a "run simulation" button which leads to simulation results page (also create endpoint there)

File: docs/view_components/project_list.rst (hunk 3 above)
Comment (mahendrark) [reply docs/view_components/project_list.rst:3]: Created an abstract view component, but I feel that the actions should be kept in the project list view component

File: docs/view_components/scenario_list.rst (hunk 4 above)
Comment (mahendrark) [reply docs/view_components/scenario_list.rst:3]: I will link the scenario abstract component to the scenario list

File: docs/view_components/project_list.rst (hunk 3 above)
Comment (Bachibouzouk) [reply docs/view_components/project_list.rst:3]: I disagree, when you have more than one project, will you duplicate "edit", "delete" etc.. or not? if yes it means the actions are tighten to the abstract component and not to the list
